stories/00-example.md

```diff
@@ -1,10 +1,9 @@
+Long term, hunger can negatively effect human beings to their utmost limits. When humans are hungry and malnourished, they may be less able to improve their livelihoods; correctly take care of their families; live complete and wholesome lives, and raise themselves out of poverty.  Children are especially prone to malnutrition in the first two years of their life cycle, and it can bring about physical and cognitive harm that diminishes future fitness, welfare, and financial well-being.
 
-Imperdiet massa tincidunt nunc pulvinar sapien et ligula ullamcorper. Massa eget egestas purus viverra accumsan in. Etiam tempor orci eu lobortis elementum nibh tellus molestie. Scelerisque varius morbi enim nunc faucibus a pellentesque sit. Eu volutpat odio facilisis mauris sit. Tristique magna sit amet purus gravida. Nisl purus in mollis nunc sed id semper risus. Malesuada proin libero nunc consequat. Lorem sed risus ultricies tristique nulla aliquet. Tellus id interdum velit laoreet id donec ultrices. Venenatis tellus in metus vulputate eu. Et malesuada fames ac turpis egestas maecenas pharetra convallis. Mi quis hendrerit dolor magna eget.
-
-Arcu felis bibendum ut tristique et egestas quis. Posuere sollicitudin aliquam ultrices sagittis orci a scelerisque purus semper. Ultricies lacus sed turpis tincidunt id aliquet risus feugiat. Elementum nisi quis eleifend quam adipiscing vitae proin sagittis. Adipiscing bibendum est ultricies integer quis auctor elit sed. Lectus sit amet est placerat in egestas erat. Diam quis enim lobortis scelerisque fermentum dui faucibus. Tincidunt praesent semper feugiat nibh sed. Porta nibh venenatis cras sed felis. Neque egestas congue quisque egestas diam. Etiam erat velit scelerisque in dictum. Sed tempus urna et pharetra pharetra. Amet venenatis urna cursus eget nunc. Cursus in hac habitasse platea dictumst quisque sagittis purus. Donec pretium vulputate sapien nec. Facilisis leo vel fringilla est ullamcorper.
+For growing nations, that is a drain on improvement with effects that can consequently last for generations. Hunger impairs a person’s capacity to be a part of an effective personnel, and make contributions to a financial boom. In the current time period, meal shortages and growing food costs can widen inequality, and result in warfare and instability.
```

Comment:
what are the political/economic causes of hunger?  could we thread through with an example?

Reply:
Added an example which details the protests in Italy, and the disturbances it can cause to politicians keeping their positions.